tests/sentry/api/endpoints/notifications/test_notification_actions_details.py

```diff
@@ -50,6 +49,48 @@ def setUp(self):
         )
         self.login_as(user=self.user)
 
+    def mock_msg_schedule_response(self, channel_id, result_name="channel"):
+        if channel_id == "channel_not_found":
+            bodydict = {"ok": False, "error": "channel_not_found"}
+        else:
+            bodydict = {
+                "ok": True,
+                result_name: channel_id,
+                "scheduled_message_id": "Q1298393284",
+            }
+        return patch(
+            "slack_sdk.web.client.WebClient.chat_scheduleMessage",
+            return_value=SlackResponse(
+                client=None,
+                http_verb="POST",
+                api_url="https://slack.com/api/chat.scheduleMessage",
+                req_args={},
+                data=bodydict,
+                headers={},
+                status_code=200,
+            ),
+        )
+
+    def mock_msg_delete_scheduled_response(self, channel_id, result_name="channel"):
+        if channel_id == "channel_not_found":
+            bodydict = {"ok": False, "error": "channel_not_found"}
+        else:
+            bodydict = {
+                "ok": True,
+            }
+        return patch(
+            "slack_sdk.web.client.WebClient.chat_deleteScheduledMessage",
+            return_value=SlackResponse(
+                client=None,
+                http_verb="POST",
+                api_url="https://slack.com/api/chat.deleteScheduledMessage",
+                req_args={},
+                data=bodydict,
+                headers={},
+                status_code=200,
+            ),
+        )
```

Comment:
I'd like to have a helper function that can reduce some of the verbosity in setting up the various mock `SlackResponse`s. At minimum, we can supply default values for the uninteresting kwargs.

It might also be a good opportunity to communicate intent. I get the intent of mocking out stuff in `slack_sdk.web.client.*` (and it's good to treat it as a black box as much as possible). However, replicating this much internal detail in the unit tests makes me a little concerned about brittleness. Abstracting it out a little would help.

Reply:
done!